build-scripts/build_llvm.py

```diff
@@ -27,11 +28,22 @@ def clone_llvm(dst_dir, llvm_repo, llvm_branch):
     return llvm_dir
 
 
-def query_llvm_version(llvm_dir):
-    GIT_LOG_CMD = f"git log --format=format:'%h' -1"
-    return subprocess.check_output(
-        shlex.split(GIT_LOG_CMD), cwd=llvm_dir, universal_newlines=True, text=True
-    )
+def query_llvm_version(llvm_info):
+    github_token = os.environ['GH_TOKEN']
+    url = f"https://api.github.com/repos/{llvm_info['owner_project']}/commits/{llvm_info['branch']}"
```

Comment:
just a minor one: `owner_project` -> `ownwer`

Reply:
the string is a combination of owner and project, isn't it?